src/cmd/initContainer.go

```diff
@@ -252,6 +252,23 @@ func initContainer(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	if utils.PathExists("/usr/lib/rpm") {
+		logrus.Debug("Configuring RPM to ignore bind mounts")
+
+		rpmConfigString := `# Written by Toolbox
+# https://github.com/containers/toolbox
+
+%_netsharedpath /dev:/media:/proc:/sys:/tmp:/var/lib/flatpak
+`
+
+		rpmConfigBytes := []byte(rpmConfigString)
+		if err := ioutil.WriteFile("/usr/lib/rpm/toolbox",
```

Comment:
I guess you should use something like `/usr/lib/rpm/macros.d/macros.toolbox` file.

Reply:
Ping, this is the naming pattern that we should follow to actually make RPM load the macros.

Reply:
Thanks for pointing that out, @praiskup !